lib/compat/wordpress-6.1/template-parts-screen.php

```diff
@@ -178,7 +178,7 @@ static function( $classes ) {
 	wp_enqueue_media();
 
 	if (
-		current_theme_supports( 'wp-block-styles' ) ||
+		current_theme_supports( 'wp-block-styles' ) &&
```

Comment:
This needs to be updated in Core in [wp-admin/site-editor.php](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/site-editor.php#L148).